setup.py

```diff
@@ -30,11 +35,10 @@ def _read_long_desc():
     'Programming Language :: Python :: Implementation :: CPython',
     'Topic :: Software Development'
   ],
-  packages=find_packages(exclude=["*.tests", "*.tests.*", "tests.*", "tests",
-      "debian"]),
-  install_requires=["pyyaml"],
+  packages=find_packages(exclude=["*.tests", "*.tests.*", "tests.*", "tests"]),
+  install_requires=_get_requirements(),
   test_suite="tests.runtests",
   entry_points={
-    "console_scripts": ["tern = tern.__init__.py:main"]
+    "console_scripts": ["tern = tern.__main__:main"]
```

Comment:
I'm still trying to get my head around how `entry_points` work. Will come back to this later.